src/modules/projects/components/tables/ProjectHouseholdsTable.tsx

```diff
@@ -32,6 +34,27 @@ const TableCellContainer = ({ children }: { children: ReactNode }) => (
   </Stack>
 );
 
+export const ASSIGNED_STAFF_COL = {
+  header: 'Assigned Staff',
+  optional: true,
+  defaultHidden: true,
+  render: (hh: HouseholdWithStaffAssignmentsFragment) => {
+    if (!hh?.staffAssignments || hh.staffAssignments.nodes.length === 0) return;
+
+    const first = hh.staffAssignments.nodes[0].user.name;
+    const rest = hh.staffAssignments?.nodesCount - 1;
```

Comment:
micro optimization, I think this is a little more concise
```suggestion
    if (!hh.staffAssignments?.nodes.length) return;

    const first = hh.staffAssignments.nodes[0].user.name;
    const rest = hh.staffAssignments.nodesCount - 1;
```